Jenkinsfile

```diff
@@ -17,15 +17,15 @@ pipeline {
         }
         stage('Build') {
             steps {
-                sh 'mvn -B -V clean verify'
+                sh 'mvn -B -V clean verify -DskipNpmConfig=false'
```

Comment:
In pom.xml this value is true. so by default it is true. What if make it default false so we don't need to set it in mvn command?

Reply:
Or can we detact the env var? If exists, run this steps otherwise not.

Reply:
The problem is we need to skip it by default, because we need to consider the local build by developer. If we set it to false by default, that means developers need to add this -DskipNpmConfig=true in their local env, which is not convenient.

Reply:
That was my concern too. If us developers don't need to add the flag but only CI need it, I am happy.

Reply:
Yeah I agree. That's why I set default to true but only add flag in CI here.